lib/v2/screens/profile_screens/guardians/guardians_tabs/guardians_screen.dart

```diff
@@ -54,12 +71,141 @@ class GuardiansScreen extends StatelessWidget {
+                          : const TabBarView(
+                              children: [
+                                MyGuardiansTab(guardianType: GuardianType.myGuardian),
+                                MyGuardiansTab(guardianType: GuardianType.imGuardian),
+                              ],
+                            )));
             })));
   }
 }
+
+void _showRecoveryStartedBottomSheet(BuildContext context, GuardianModel guardian) {
+  showModalBottomSheet<void>(
+    context: context,
+    builder: (BuildContext context) {
+      return Container(
+        child: Padding(
+          padding: const EdgeInsets.only(top: 16.0, bottom: 16),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisSize: MainAxisSize.min,
+            children: <Widget>[
+              Center(
+                child: Container(
+                  child: const SizedBox(height: 2, width: 40),
+                  color: Colors.black,
+                ),
+              ),
+              const SizedBox(height: 20),
+              Padding(
+                padding: const EdgeInsets.only(left: 16.0, right: 16.0),
+                child: Center(
+                    child: Text(
+                  "A motion to Recover your Key has been initiated by ${guardian.nickname}",
+                  style: const TextStyle(color: Colors.black, fontSize: 16),
+                )),
+              ),
+              const SizedBox(height: 20),
+              TextButton.icon(
+                onPressed: () {
+                  _showStopRecoveryConfirmationDialog(guardian, context);
+                },
+                label: const Text(
+                  "Stop this Recovery",
+                  style: TextStyle(color: Colors.blue),
+                ),
+                icon: const Icon(Icons.cancel_rounded, color: AppColors.darkGreen3),
+              ),
+            ],
+          ),
+        ),
+      );
+    },
+  );
+}
+
+void _showStopRecoveryConfirmationDialog(GuardianModel guardian, BuildContext context) {
+  showDialog(
+    context: context,
+    builder: (BuildContext context) {
+      return AlertDialog(
+        content:
+            const Text("Are you sure you want to stop key recovery process", style: TextStyle(color: Colors.black)),
+        actions: [
+          TextButton(
+            child: const Text('No: Dismiss'),
+            onPressed: () {
+              Navigator.pop(context);
+            },
+          ),
+          TextButton(
+            child: const Text("Yes: Stop Key Recovery"),
+            onPressed: () {
+              BlocProvider.of<GuardiansBloc>(context).add(OnStopRecoveryForUser());
+              Navigator.pop(context);
+              Navigator.pop(context);
+            },
+          )
+        ],
+      );
+    },
+  );
+}
+
+void _showRemoveGuardianDialog(BuildContext buildContext, GuardianModel guardian) {
```

Comment:
these dialogs dont follow our current dialogs. 
We dont have designs for the guardians dialogs in figma. I should ask Jason for designs. 

Right now the functionality is in place. We will update the dialogs later when we get designs.